### PR TITLE
devops: migrate electron & video bots to re-usable run-test action

### PR DESCRIPTION
**Pull request by mxschmitt**

### CODE REVIEW

File: .github/workflows/tests_secondary.yml [hunk 1]
```diff
@@ -229,7 +229,7 @@ jobs:
         PWTEST_CHANNEL: chrome
 
   chromium_tot:
-    name: Chromium TOT ${{ matrix.os }}
```
Comment (mxschmitt) [inline .github/workflows/tests_secondary.yml:232]: drive-by: so we can just grep for tip-of-tree when rolling ToT. Happy to do it the other way around as well. Right now some bots have ToT in their name, some tip-of-tree.

**Review by yury-s — APPROVED** [review]
